src/main/java/com/northconcepts/datapipeline/foundations/examples/jdbc/GenerateJavaBeansFromADatabase.java

```diff
@@ -0,0 +1,68 @@
+package com.northconcepts.datapipeline.foundations.examples.jdbc;
+
+import com.northconcepts.datapipeline.foundations.jdbc.JdbcConnection;
+import com.northconcepts.datapipeline.foundations.jdbc.JdbcTable;
+import com.northconcepts.datapipeline.foundations.jdbc.JdbcTableColumn;
+import com.northconcepts.datapipeline.foundations.sourcecode.CodeWriter;
+import com.northconcepts.datapipeline.foundations.sourcecode.JavaCodeBuilder;
+
+
+public class GenerateJavaBeansFromADatabase {
+
+    static JavaCodeBuilder code = new JavaCodeBuilder();
+    static CodeWriter sourceWriter = code.getSourceWriter();
+
+    public static void main(String... args) {
+
+        JdbcConnection connection = new JdbcConnection()
+                .setDriverClassName("org.postgresql.Driver")
+                .setUrl("jdbc:postgresql://localhost:5432/customers")
+                .setUsername("username")
+                .setPlainTextPassword("password");
+
+
+        for(JdbcTable table : connection.loadTables().getTables()) {
+            createTableClass(table);
+        }
+
+        System.out.println(code.getSource());
+    }
+
+    public static void createTableClass(JdbcTable table) {
+        sourceWriter.println("public class %s {", table.getName());
+        sourceWriter.indent();
+        sourceWriter.println();
+        for(JdbcTableColumn column : table.getColumns()) {
+            sourceWriter.println("private %s %s;", column.getMethodSuffix(), column.getName());
```

Comment:
@Bryan-Noel is `column.getMethodSuffix()` the right thing to use here?  Does it handle date and time columns well?

Reply:
@DeleTaylor , just tested this with Date, Time and Timestamp columns and it does handle those types well.